### PR TITLE
JADE: Play background music on main menu

### PR DESCRIPTION
**Pull request by dressupgeekout**
I mean, it works, I dunno :}

### CODE REVIEW

**Comment by DrMcCoy** [comment]
Hmm, could you rather save that ChannelHandle and use `SoundMan.stopChannel()`? Put this into their own methods, `playMenuMusic()` and `stopMenuMusic` too, please.

I.e. similar to how KotOR does it: https://github.com/xoreos/xoreos/blob/master/src/engines/kotor/game.cpp#L105

**Comment by dressupgeekout** [comment]
Oh heh that's definitely cleaner.

**Comment by dressupgeekout** [comment]
OK let's try this. I decided that `Jade::Game::playMenuMusic()` shouldn't accept a parameter to  let the caller decide _which_ menu music to play, because, unlike KOTOR, there's only one menu music (even during character creation and stuff like that)

File: src/engines/jade/game.cpp [hunk 1]
```diff
@@ -100,6 +102,17 @@ void Game::runModule() {
 	_module->clear();
 }
 
+void Game::playMenuMusic()
+{
```
Comment (DrMcCoy) [inline src/engines/jade/game.cpp]: That opening brace needs to go into the previous line, please.

File: src/engines/jade/game.cpp [hunk 2]
```diff
@@ -100,6 +102,17 @@ void Game::runModule() {
...
+	stopMenuMusic();
+	_menuMusic = playSound("background", Sound::kSoundTypeMusic, true);
+}
+
+void Game::stopMenuMusic()
+{
```
Comment (DrMcCoy) [inline src/engines/jade/game.cpp]: Here too

**Comment by dressupgeekout** [comment]
OK, here we go

**Comment by DrMcCoy** [comment]
Merged as b9555a11844a5e0cde6641b4887c90961f01ecf9, thanks! :)

**Pull request closed without merging**